hack/deploy.sh

```diff
@@ -28,9 +28,13 @@ do
 
   # be verbose on last iteration only
   if [[ $iterations -eq $((max_iterations - 1)) ]] || [[ -n "${VERBOSE}" ]]; then
-    ${OC_TOOL} kustomize $feature_dir | envsubst | ${OC_TOOL} apply -f -
+    # oc / kubectl reject multiple manifests now as soon as one "kind" in them does not exist yet
+    # so we need to apply one manifest by one
+    # since xargs' delimiter is limited to one char only or a control code, we replace the manifest delimiter "---"
+    # with a "vertical tab (\v)", which should never be used in (at least our) manifests.
```

Comment:
Please mention the kubernetes issue here to make it clear this is a workaround for a bug only.

Reply:
done